lib/rubocop/cop/style/extra_spacing.rb

```diff
@@ -67,11 +67,14 @@ def check_tokens(ast, t1, t2)
         end
 
         def check_assignment(token)
-          # minus 2 is because pos.line is zero-based
-          line = processed_source.lines[token.pos.line - 2]
+          preceding = @asgn_lines.include?(token.pos.line - 1)
+          line = if preceding
```

Comment:
Yeah. I know the name was there before your PR, but I feel this is an opportunity to improve it.